doc/UsersGuide/source/BuildingAndRunning.rst

```diff
@@ -86,9 +85,8 @@ that, these environment variables need to be set, as shown in :numref:`Table %s
    +------------------+-----------------------------------------------------------------+
    | **NCEP Library** | **Environment Variables**                                       |
    +==================+=================================================================+
-   |  nemsio          | export NEMSIO_INC=<path_to_nemsio_include_dir>                  |
-   +------------------+-----------------------------------------------------------------+
-   |                  | export NEMSIO_LIB=<path_to_nemsio_lib_dir>/libnemsio<version>.a |
+   |  nemsio          || export NEMSIO_INC=<path_to_nemsio_include_dir>                 |
+   |                  || export NEMSIO_LIB=<path_to_nemsio_lib_dir>/libnemsio<version>.a|
    +------------------+-----------------------------------------------------------------+
    |  bacio           | export BACIO_LIB4=<path_to_bacio_lib_dir>/libbacio<version>.a   |
    +------------------+-----------------------------------------------------------------+
```

Comment:
I think these are not necessary anymore.

Reply:
Tables 3.1 and 3.2 are not required? Taken care of by CMake on Tier-1 platforms?

Reply:
I'm not sure what are tables 3.1 and 3.2. 
CMake will use nceplib's  config files. We do not use these environment variables anymore.